src/contracts/Staking.sol

```diff
@@ -116,14 +126,27 @@ contract Staking is Ownable {
     }
     Epoch public epoch;
 
+    struct Claim {
+        uint256 amount;
+        uint256 gons;
+        uint256 expiry;
+        bool lock; // prevents malicious delays
+    }
+    mapping(address => Claim) public warmupInfo;
```

Comment:
same minor nit picks on the naming convention, i think we should do a find replace and make all of these `warmUpInfo` `coolDownInfo` to stick with strict camelCase.